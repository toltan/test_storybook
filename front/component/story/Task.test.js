import initStoryshots from "../../../node_modules/@storybook/addon-storyshots";
initStoryshots();

// import "../../../node_modules/@jest/globals";
// import Task from "./Task";

// describe("jest 動作確認", () => {
//   test("true toBeTruthy", () => {
//     expect(true).toBeTruthy();
//   });
// });
