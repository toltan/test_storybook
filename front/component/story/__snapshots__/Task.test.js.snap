// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Component/Story/Task Archived 1`] = `
<div>
  Story
</div>
`;

exports[`Storyshots Component/Story/Task Default 1`] = `
<div>
  Default
</div>
`;

exports[`Storyshots Component/Story/Task Pinned 1`] = `
<div>
  Pinned
</div>
`;

exports[`Storyshots Component/Story/Task Story 1`] = `
<div>
  Story
</div>
`;

exports[`Storyshots Task Archived 1`] = `
<div
  className="list-item"
>
  id: 3
  , 
  message: Story
</div>
`;

exports[`Storyshots Task Default 1`] = `
<div
  className="list-item"
>
  id: 1
  , 
  message: Default!
</div>
`;

exports[`Storyshots Task Pinned 1`] = `
<div
  className="list-item"
>
  id: 2
  , 
  message: Pinned
</div>
`;

exports[`Storyshots Task Story 1`] = `
<div
  className="list-item"
>
  id: 4
  , 
  message: Story
</div>
`;

exports[`Storyshots TaskList Default 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="list-item"
    >
      id: 0
      , 
      message: Default!
    </div>
    <div
      className="list-item"
    >
      id: 1
      , 
      message: Pinned
    </div>
    <div
      className="list-item"
    >
      id: 2
      , 
      message: { ...TaskStories.Default.args.task, id: '3', title: 'Task 3' }
    </div>
    <div
      className="list-item"
    >
      id: 3
      , 
      message: { ...TaskStories.Default.args.task, id: '4', title: 'Task 4' }
    </div>
    <div
      className="list-item"
    >
      id: 4
      , 
      message: { ...TaskStories.Default.args.task, id: '5', title: 'Task 5' }
    </div>
  </div>
</div>
`;

exports[`Storyshots TaskList Empty 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="wrapper-message"
    >
      <span
        className="icon-check"
      />
      <div
        className="title-message"
      >
        You have no tasks
      </div>
      <div
        className="subtitle-message"
      >
        Sit back and relax
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots TaskList Loading 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
  </div>
</div>
`;

exports[`Storyshots TaskList With Pinned Tasks 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="list-item"
    >
      id: 0
      , 
      message: Default!
    </div>
    <div
      className="list-item"
    >
      id: 1
      , 
      message: Pinned
    </div>
    <div
      className="list-item"
    >
      id: 2
      , 
      message: { ...TaskStories.Default.args.task, id: '3', title: 'Task 3' }
    </div>
    <div
      className="list-item"
    >
      id: 3
      , 
      message: { ...TaskStories.Default.args.task, id: '4', title: 'Task 4' }
    </div>
    <div
      className="list-item"
    >
      id: 4
      , 
      message: { ...TaskStories.Default.args.task, id: '5', title: 'Task 5' }
    </div>
    <div
      className="list-item"
    >
      id: 5
      , 
      message: { id: 6, message: 'With Pinned Tasks' }
    </div>
  </div>
</div>
`;
